src/spicy/spicyz/CMakeLists.txt

```
@@ -3,7 +3,7 @@
 configure_file(config.h.in config.h)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/config.h DESTINATION include/zeek/spicy/spicyz)
 
-add_executable(spicyz driver.cc glue-compiler.cc main.cc)
+add_executable(spicyz driver.cc glue-compiler.cc zeek-version.cc main.cc)
```

Lets maintain alphabetical order


```suggestion
add_executable(spicyz driver.cc glue-compiler.cc main.cc zeek-version.cc)
```